### DISCUSSION TITLE
io/ioutil deprecated

### DISCUSSION THREAD

**Discussion by ErikEk** [lightningnetwork/lnd#7750] [Ideas]
Hello,
It seems like the io/ioutil package has been deprecated since go 1.16 but I find it everywhere in the code base.
[io/ioutil](https://pkg.go.dev/io/ioutil)
Shouldn't we repleace it?
Related issues: https://github.com/lightningnetwork/lnd/issues/6720

**Comment by guggero** [comment]
Are there any plans for removing them from the old `io/ioutil` package in any upcoming version of Go? If yes, then we should probably replace them soon. If not, then I don't think this is very urgent. But since there is a drop-in replacement this is also low risk as it's really just search/replace. So feel free to get a PR started.

**Comment by ErikEk** [comment]
They will not remove the package or those functions from io/ioutil anytime soon according to the release notes.
I will add a PR with the changes listed in the release note link below:
https://go.dev/doc/go1.16#:~:text=FS%20implementations.-,Deprecation%20of%20io%2Fioutil,been%20moved%20to%20other%20packages.
